chrome/locale/en-US/zotero/zotero.properties

```diff
@@ -359,6 +359,8 @@ pane.items.menu.createParent              = Create Parent Item…
 pane.items.menu.createParent.multiple     = Create Parent Items
 pane.items.menu.renameAttachments = Rename File from Parent Metadata
 pane.items.menu.renameAttachments.multiple = Rename Files from Parent Metadata
+pane.items.menu.duplicateAndConvert.toBookSection = Create Book Section
+pane.items.menu.duplicateAndConvert.toBook = Create Book
```

Comment:
I think this one has to be "Create Book Item", even if it's not quite symmetrical. "Create Book Section" on its own seems fine.

Reply:
Or "Create Book from Book Section"?

Reply:
I think "Create Book Section" and "Create Book from Book Section" would work.

Reply:
Sure, pushed the latter option.